programming/sublime-text/files/sublime_text.desktop

```diff
@@ -0,0 +1,23 @@
+[Desktop Entry]
```

Comment:
As far as I can tell you seem to have an exact copy of the desktop file that is also included in the archive here.
Any particular reason for not simply using the latter?